src/test/java/com/github/martapietka/ppm2pbm/RgbToGrayscaleConverterTest.java

```diff
@@ -3,9 +3,9 @@
 import static org.assertj.core.api.Assertions.*;
 import org.junit.jupiter.api.Test;
 
-public class rgbToVvalueTest {
+public class RgbToGrayscaleConverterTest {
     @Test
-    void convertRgbToVvalueGreen() {
+    void convertRgbToGreen() {
```

Comment:
Zmieniłam też nazwy wszystkich testów, bo wydaje mi się, że są bardziej czytelne teraz ;)

Reply:
To dobry pomysł, natomiast proponowałbym nazewnictwo w rodzaju `convertRgbToGrayscaleFromGreen`, bo teraz nazwa jest trochę myląca 😉 Natomiast wstrzymajmy się z tym, pokażę Ci jeszcze jak można do tego podejść inaczej.